src/python/src/grpc/_adapter/_c/types.c

```diff
@@ -31,19 +31,30 @@
  *
  */
 
-#ifndef _ADAPTER__COMPLETION_QUEUE_H_
```

Comment:
"src/python/src/grpc/_adapter/_completion_queue.h → src/python/src/grpc/_adapter/_c/types.c" makes for another funny diff.